tutorials/best_practices/scene_organization.rst

```diff
@@ -254,7 +254,7 @@ Choosing a node tree structure
 So, a developer starts work on a game only to stop at the vast possibilities
 before them. They might know what they want to do, what systems they want to
 have, but *where* to put them all? Well, how one goes about making their game
-is always up to them. One can construct node trees in a myriad of ways.
+is always up to them. One can construct node trees in myriad ways.
```

Comment:
That change seems to be up for debate: https://www.dictionary.com/e/how-do-you-use-the-word-myriad/

Let's use `countless` instead then?

Reply:
Yeah, lets go with that.